OasisDEX/oasis-borrow
# Lw/7575 follow guni

# [Follow/unfollow earn vaults](https://app.shortcut.com/oazo-apps/story/7575/follow-unfollow-support-earn-vaults)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
- Added follow/unfollow support for earn vaults
  
## How to test 🧪
- FollowVaults flag must be on
- Open Earn vault that exist in db
- 
![image](https://user-images.githubusercontent.com/6871923/213245447-28d260ce-c9c0-418d-9503-b296da0d7151.png)

- follow vault, verify db if record added, check if button state changed to following
![image](https://user-images.githubusercontent.com/6871923/213245622-df80213d-125e-4b80-8662-f4b394bd1923.png)

- Unfollow, check if recorded was erased from db
![image](https://user-images.githubusercontent.com/6871923/213245763-a1490690-f9ce-4157-8ef2-eff9744e631a.png)
